src/extension.ts

```diff
@@ -87,16 +97,53 @@ export async function activate(context: ExtensionContext) {
     .catch(err => {
       const message =
         "Unable to find a Java 8 or Java 11 installation on this computer. " +
-        "To fix this problem, update the 'Java Home' setting to point to a Java 8 or Java 11 home directory";
+        "To fix this problem, update the 'Java Home' setting to point to a Java 8 or Java 11 home directory " +
+        "or select a version to install automatically (window will need to be reloaded afterwards)";
```

Comment:
nit: instead of writing it here, can we automatically prompt the user to reload the window after installation?

Reply:
The user will be prompted - it's just to warn them about it